examples/oneapi/cpp/source/decision_forest/df_reg_dense_batch.cpp

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "oneapi/dal/algo/decision_forest.hpp"
+#include "oneapi/dal/data/accessor.hpp"
```

Comment:
You can delete 
```c++
#include "oneapi/dal/data/accessor.hpp"
#include "oneapi/dal/data/table.hpp"
```

Because they included in "example_util/utils.hpp"

Reply:
applied